### PR TITLE
Ensure line chart height and width are positive

### PR DESCRIPTION
**Pull request by follan**
#307

### CODE REVIEW

File: src/components/LineChart/index.js [hunk 1]
```diff
@@ -266,12 +266,14 @@ const LineChart = props => {
...
+    getYAxisCollectionWidth(AxisPlacement.LEFT, props) -
+    getYAxisCollectionWidth(AxisPlacement.RIGHT, props);
+  const chartHeight = height - getXAxisHeight(xAxisHeight) - contextChartSpace;
   const chartSize = {
-    width:
-      width -
-      getYAxisCollectionWidth(AxisPlacement.LEFT, props) -
-      getYAxisCollectionWidth(AxisPlacement.RIGHT, props),
-    height: height - getXAxisHeight(xAxisHeight) - contextChartSpace,
+    width: chartWidth > 0 ? chartWidth : 0,
```
Comment (evancharlton) [inline src/components/LineChart/index.js]:
```suggestion
    width: Math.max(0, chartWidth),
```

File: src/components/LineChart/index.js [hunk 2]
```diff
@@ -266,12 +266,14 @@ const LineChart = props => {
...
+    height: chartHeight > 0 ? chartHeight : 0,
```
Comment (evancharlton) [inline src/components/LineChart/index.js]:
```suggestion
    height: Math.max(0, chartHeight),
```

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-310.surge.sh

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-310.surge.sh

**Comment by evancharlton** [comment]
(Thanks for tracking this down! This has bugged me for a while, but I was always too scared to try fixing it again)

**Comment by evancharlton** [comment]
Final nit: when you squash & merge, could you write a decent commit message about what this is doing and why?

**Pull request merged**